turtle_tf2_py/setup.py

```diff
@@ -0,0 +1,31 @@
+import os
+from glob import glob
+from setuptools import setup
+
+package_name = 'turtle_tf2_py'
+
+setup(
+    name=package_name,
+    version='0.0.0',
+    packages=[package_name],
+    data_files=[
+        ('share/ament_index/resource_index/packages', ['resource/' + package_name]),
+        ('share/' + package_name, ['package.xml']),
+        (os.path.join('share', package_name, 'launch'), glob(os.path.join('launch', '*.launch.py'))),
+        (os.path.join('share', package_name, 'rviz'), glob(os.path.join('rviz', '*.rviz'))),
+    ],
+    install_requires=['setuptools'],
+    zip_safe=True,
+    maintainer='Shyngyskhan Abilkassov',
+    maintainer_email='abilkasov@gmail.com',
+    description='turtle_tf2_py demonstrates how to write a ROS2 Python tf2 broadcaster and listener with the turtlesim. The turtle_tf2_listener commands turtle2 to follow turtle1 around as you drive turtle1 using the keyboard.',
```

Comment:
Does the linter complain about this line? It might be nice to wrap the line.

I think that you should add tests to this packages that run the linter, check copyrights, and perhaps check docstrings. You can find an example here:
https://github.com/ros2/demos/tree/master/demo_nodes_py/test

Reply:
How do I run the tests after I add those files? Should I also insert `test_depend` tags and see how it builds?

Reply:
You can do the same `test_depend`s as in the demos:
https://github.com/ros2/demos/blob/3af19c93c7abd9a83caca47de0780e941faeb22b/demo_nodes_py/package.xml#L20-L23

To run them, you can use `colcon test --packages-select <packages you want to test...>`

Reply:
Thanks, fixed now